src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/EventHandlerModifier/EventHandlerModifierFixerTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Acuminator.Analyzers.StaticAnalysis.EventHandlerModifier;
+using Acuminator.Analyzers.StaticAnalysis.PrivateEventHandlers;
+using Acuminator.Analyzers.StaticAnalysis.PXGraph;
+using Acuminator.Tests.Helpers;
+using Acuminator.Tests.Verification;
+using Acuminator.Utilities;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Xunit;
+
+namespace Acuminator.Tests.Tests.StaticAnalysis.EventHandlerModifier
+{
+	public class EventHandlerModifierFixerTests : CodeFixVerifier
+	{
+		protected override CodeFixProvider GetCSharpCodeFixProvider() => new EventHandlerProtectedModifierFix();
+
+		protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() => new PXGraphAnalyzer(
+				CodeAnalysisSettings.Default
+									.WithRecursiveAnalysisEnabled()
+									.WithStaticAnalysisEnabled()
+									.WithSuppressionMechanismDisabled(),
+				new EventHandlerModifierAnalyzer());
+
+
+		[Theory]
+		[EmbeddedFileData("InvalidHandlerModifier.cs",
+						  "InvalidHandlerModifier_Expected.cs")]
+		public void Test_Invalid_Modifiers_In_Event_Handlers(string actual, string expected)
```

Comment:
The test method should return `Task`, await `VerifyCSharpFixAsync` and be async methods

Reply:
Done.